app.py

```diff
@@ -111,6 +111,9 @@ def update_cfg_by_env(cfg):
         if item[1]:
             cfg['common'][item[0]] = item[1]
 
+    if cfg.get('repo_lists') is None:
+        cfg['repo_lists'] = {}
```

Comment:
Nit: `<dict>.setdefault(k, v)` is easier to read.

Reply:
The same syntax was used a few lines earlier, so I prefer to leave it as it is. In the future I will prefer to use `setdefault`.